### ISSUE TITLE
something about sampling with what model,condition?and uncondition?

### ISSUE THREAD

**Issue by xiaoningpi** [Y-debug-sys/Diffusion-TS#30]
I am currently training my dataset with this code and have a few questions I don't understand:
1. In the readme, there is only one training command, but there are three types of sampling: Unconstrained,imputation,Forecasting.
  To train conditional diffusion models and unconditional diffusion models, it is necessary to set the training set split ratio to less than 1 or equal to 1. Besides changing the training set split ratio, are there any other parameters that need to be modified?
2. How do I generate samples without  conditions? Is it by using the command :python main.py --name {name} --config_file {config.yaml} --gpu 0 --sample 0 --milestone {checkpoint_number}?
4. Does the imputation task require training a conditional model?

**Comment by Y-debug-sys** [comment]
Hi, my corresponding answers are as follows:
1. The split is for testing purposes, as the prediction and imputation require a test set, while validating unconditional generation just requires the training set. The parameters depend on your data. But please be cautious about the model size problem, for example, in previous experiments, `d_model` has never exceeded 128, and `n_layer_enc` and `n_layer_dec` have never been more than 4.
2. Yes, you're right. Its generations will be saved into a .npy file, then you can compare their similarity to the original data, using codes in [metric_pytorch.ipynb](https://github.com/Y-debug-sys/Diffusion-TS/blob/main/Experiments/metric_pytorch.ipynb), [metric_tensorflow.ipynb](https://github.com/Y-debug-sys/Diffusion-TS/blob/main/Experiments/metric_tensorflow.ipynb) and [Tutorial_0.ipynb](https://github.com/Y-debug-sys/Diffusion-TS/blob/main/Tutorial_0.ipynb).
3. No, a pre-trained unconditional diffusion model is all you need.

Thanks!
